acceptance/update_test.go

```diff
@@ -127,7 +133,7 @@ func testRaftUpdateInner(t *testing.T, c cluster.Cluster, cfg cluster.TestConfig
 	}
 
 	// Unfreezing again should be a no-op.
-	if reply, err := postFreeze(c, false); err != nil {
+	if reply, err := postFreeze(c, false, long); err != nil {
```

Comment:
I want only the command which I'm expecting to time out to actually time out (but I'm not ready to wait a minute). For everything else, I want to reduce spurious errors, so a long timeout.